packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -2381,6 +2397,17 @@ You can publish a theme for a page or email template with different combinations
 | `OKTA_DEFAULT`                  | Use the Okta logo and the Okta colors in the email templates.    |
 | `FULL_THEME`                    | Use the logo from Theme and `primaryColorHex` as the background color for buttons.      |
 
+<ApiLifecycle access="ea" />
+
+> **Note:** Okta loading page variants are gated behind the **Loading Page Animation** Early Access feature and must be enabled. See [Feature Lifecycle Management](/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features).
+
+#### Variants for the Okta loading page:
```

Comment:
```suggestion
#### Variants for the Okta loading page:

<ApiLifecycle access="ea" />

> **Note:** Okta loading page variants are only available if the **Loading Page Animation** Early Access feature is enabled. See [Feature Lifecycle Management](/docs/concepts/feature-lifecycle-management/) and [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features).
```
Add the EA flag under the section heading since the feature applies to that section, not the previous one.